manticore/core/manticore.py

```diff
@@ -648,33 +506,6 @@ def is_shutdown(self):
         ''' Returns True if shutdown was requested '''
         return self._executor.is_shutdown()
 
-    #############################################################################
-    #############################################################################
-    #############################################################################
-    # Move all the following elsewhere Not all manticores have this
-    def _get_symbol_address(self, symbol):
```

Comment:
Although it is not used anywhere, I have moved it to `native/manticore.py:Manticore`.